src/Raven.Client/Http/ServerNode.cs

```diff
@@ -84,15 +84,21 @@ public void DiscardServerVersion()
             _lastServerVersionCheck = 0;
         }
 
-        internal static List<ServerNode> CreateFrom(ClusterTopology topology)
+        internal static Topology CreateFrom(ClusterTopology topology, long etag)
         {
-            var nodes = new List<ServerNode>();
+            var newTopology = new Topology()
+            {
+                Etag = etag,
+                Nodes = new List<ServerNode>(),
+                Promotables = new List<ServerNode>()
```

Comment:
Is there any scenario where we would want to populate this? Are there any `SelectedNodeTag` requests of server type that would use the `ClusterRequestExecutor` in the *server*?